ui/app/adapters/pki/config.js

```diff
@@ -0,0 +1,22 @@
+import { encodePath } from 'vault/utils/path-encoding-helpers';
+import ApplicationAdapter from '../application';
+
+export default class PkiConfigAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  urlForCreateRecord(modelName, snapshot) {
+    const { backend, formType, type } = snapshot.record;
+    if (!backend || !formType) {
+      throw new Error('URL for create record is missing required attributes');
+    }
+    const baseUrl = `${this.buildURL()}/${encodePath(backend)}/issuers`;
```

Comment:
I'm confused why `issuers` is here - since this is the config I thought the URL would be `pki/root/generate/:type` or `pki/intermediate/generate/:type` 🤔 But I have no idea 🥴